internals/overlord/servstate/request.go

```diff
@@ -4,28 +4,85 @@ import (
 	"fmt"
 
 	"github.com/canonical/pebble/internals/overlord/state"
+	"github.com/canonical/pebble/internals/plan"
 )
 
 // ServiceRequest holds the details required to perform service tasks.
 type ServiceRequest struct {
 	Name string
 }
 
+func get_or_create_lane(s *state.State, service *plan.Service, service_lane_mapping map[string]int) int {
```

Comment:
Let's just make sure we use Go naming conventions throughout, for example `getOrCreateLane` -- similar below.